scripts/example-pipelines-test.sh

```diff
@@ -1,3 +1,166 @@
 #!/bin/bash
 
-echo "Temporary test file"
+cp -L $KUBECONFIG /tmp/kubeconfig && export KUBECONFIG=/tmp/kubeconfig
+export IMG=${CI_OPERATOR_IMG}
+
+winImageDownloadURL="http://http-server/downloads/disk.img"
+
+# Deploy resources
+echo "Deploying resources"
+./scripts/deploy-resources.sh
+
+# SECRET
+key="/tmp/secrets/accessKeyId"
+token="/tmp/secrets/secretKey"
+
+if test -f "$key" && test -f "$token"; then
+  id=$(cat $key | tr -d '\n' | base64)
```

Comment:
What about this?

Reply:
I have used same names for these variables like in common templates repo from where is this part of code inspired. If you think it would be useful to rename it I will do it.